ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/wholeBodyHardwareControl/AvatarMultiThreadingFactory.java

```diff
@@ -167,7 +165,7 @@ public void join()
 
    public void stop()
    {
-      LogTools.info("Calling shutdown in the controller factory");
+      System.out.println("Calling stop in the multi-threading factory");
```

Comment:
why would you change this from a LogTools call? You lose the stack trace using println, which makes it next to impossible to find where it's printing from.

Reply:
That's correct, it won't print at all with LogTools, but you could add "(AvatarMultiThreadingFactory.java:168):" manually